io.github.liberodark.OpenDrive.json

```diff
@@ -0,0 +1,60 @@
+{
+  "app-id": "io.github.liberodark.OpenDrive",
+  "base": "io.atom.electron.BaseApp",
+  "base-version": "18.08",
+  "command": "odrive",
+  "rename-desktop-file": "odrive.desktop",
+  "rename-icon": "odrive",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "separate-locales": false,
+  "finish-args": [
+    "--device=all",
+    "--device=dri",
+    "--env=XDG_CURRENT_DESKTOP=Unity",
+    "--filesystem=home",
```

Comment:
Is it a bad thing to remove this access and say, for instance: `--persist=~/OpenDrive/`? That would make that path the default for this application...but then whoever doesn't like it would be mad :rofl:

Reply:
I consider this a good thing, yet I'm not affiliated with Flathub.
User can always grant permissions to whatever they want, this shouldn't be by default.

Reply:
No, and it's actually preferred when it makes sense. I'm not sure if it does for ODrive, given sandbox' `~/ODrive` won't be the one outside it.